### PR TITLE
Initial Hindi translation.

### PR DESCRIPTION
**Pull request by Scrambled777**
@blissd 

This PR adds initial Hindi `hi` translation.

Query regarding following strings:

1. Highlighted strings were not translated:

Screenshot_20240524_142516

2. Could you also add strings from this window for translation:

Screenshot_20240524_142601

### CODE REVIEW

File: i18n/hi/fotema.ftl [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+   [4] अप्रैल {$year}
+   [5] मई {$year}
+   [6] जून {$year}
+   [7] जुलाई {$year}
+   [8] अगस्त {$year}
+   [9] सितम्बर {$year}
+   [10] अक्टूबर {$year}
+   [11] नवम्बर {$year}
+   [12] दिसम्बर {$year}
+  *[अन्य] {$year}
```
Comment (blissd) [inline i18n/hi/fotema.ftl:55]: The text between the square brackets should not be translated and should remain as `other`.

File: i18n/hi/fotema.ftl [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+  }
+}
```
Comment (blissd) [inline i18n/hi/fotema.ftl:57]: This is an extra closing brace which also exists in `en-US/fotema.ftl`  (which I'll fix shortly). When I remove the extra brace each `fotema.ftl` file and fix the naming of the default case, then month labels translate correctly.

**Review by blissd — CHANGES_REQUESTED** [review]
Please rename default case back to 'other' and remove extra closing brace.

File: i18n/hi/fotema.ftl (hunk 2 above)
Comment (blissd) [reply i18n/hi/fotema.ftl:57]: I've pushed a fix to `main` to remove the extra closing brace in `i18n/en-US/fotema.ftl`.

**Comment by blissd** [comment]
Hmmm... as for translating the "About" dialog... I see that I've made a problem for myself by not using `gettext` :cry:. I've used the standard libadwaita AboutDialog and don't have access to those label strings to create message IDs for them so I can't currently make them translatable with Fluent. I'll have to have a think as to whether I revist trying to get `gettext` to work (which I don't really want to), or if there is another solution :-/

File: i18n/hi/fotema.ftl (hunk 1 above)
Comment (Scrambled777) [reply i18n/hi/fotema.ftl]: I have updated fml file accordingly. Thanks.

File: i18n/hi/fotema.ftl (hunk 2 above)
Comment (Scrambled777) [reply i18n/hi/fotema.ftl:56]: Fixed in translated file.

**Comment by Scrambled777** [comment]
> Please rename default case back to 'other' and remove extra closing brace.

Done. 

> Hmmm... as for translating the "About" dialog... I see that I've made a problem for myself by not using `gettext` 😢. I've used the standard libadwaita AboutDialog and don't have access to those label strings to create message IDs for them so I can't currently make them translatable with Fluent. I'll have to have a think as to whether I revist trying to get `gettext` to work (which I don't really want to), or if there is another solution :-/

Could I suggest getting help from Gnome i18n [Matrix channel](https://matrix.to/#/#i18n:gnome.org)?

**Comment by blissd** [comment]
> > Please rename default case back to 'other' and remove extra closing brace.
> 
> Done.
> 
> > Hmmm... as for translating the "About" dialog... I see that I've made a problem for myself by not using `gettext` 😢. I've used the standard libadwaita AboutDialog and don't have access to those label strings to create message IDs for them so I can't currently make them translatable with Fluent. I'll have to have a think as to whether I revist trying to get `gettext` to work (which I don't really want to), or if there is another solution :-/
> 
> Could I suggest getting help from Gnome i18n [Matrix channel](https://matrix.to/#/#i18n:gnome.org)?

I suspect I will scolded for not using gettext 😆

**Comment by Scrambled777** [comment]
> @Scrambled777 Would you like a translator credit in the about page? I'm happy to add a name and email, or a github username, or whatever you like to be identified by online.

@blissd, Could you add `Scrambled777` `weblate.scrambled777@simplelogin.com`?
My handle & Email.

Cheers.

**Pull request merged**